okhttp/src/main/java/okhttp3/internal/platform/Platform.kt

```diff
@@ -126,7 +126,7 @@ open class Platform {
     socket.connect(address, connectTimeout)
   }
 
-  open fun log(level: Int, message: String, t: Throwable?) {
+  open fun log(level: Int = INFO, message: String, t: Throwable? = null) {
```

Comment:
Put message first so you can safely omit level?

Reply:
Sounds good.